core/src/main/java/org/opensearch/sql/executor/streaming/Batch.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.executor.streaming;
+
+import java.util.List;
+import lombok.Data;
+import org.opensearch.sql.storage.split.Split;
+
+/**
+ * A batch of streaming execution.
+ */
+@Data
+public class Batch {
+  private final List<Split> splits;
```

Comment:
For FS, because split-to-files is 1 to N and there is always single split here, wondering in which case a batch includes more than one split?

Reply:
good catch. batch should only contain 1 split.
done.